lib/internal/bootstrap/pre_execution.js

```diff
@@ -477,8 +480,9 @@ function initializePolicy() {
         throw new ERR_MANIFEST_ASSERT_INTEGRITY(manifestURL, realIntegrities);
       }
     }
-    require('internal/process/policy')
-      .setup(src, manifestURL.href);
+    policy.setup(src, manifestURL.href);
+  } else {
+    policy.setup(null);
```

Comment:
Can you add some comments to explain when we'd need to deal with an empty policy here and below?

Reply:
How about something like:

```mjs
// this disables the policy TDZ checks so code can just check if (!policy.manifest)
// it is important to keep the policy in TDZ until the user supplied
// options are determined at startup
```

Or is that too much?

and after these 2 if/else:

```mjs
// ensure that policy no longer has a TDZ by checking
// that we can access .manifest
policy.manifest
```

Reply:
Sounds good, thanks

Reply:
added, did not add to the worker_threads bootstrap though, we could make a shared fn but I feel like duplication is fine since they populate the actual manifest if specified somewhat differently.